packages/opentelemetry-semantic-conventions/src/trace/os.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The operating system (OS) on which the process represented by this resource is running.
+ *
+ * In case of virtualized environments, this is the operating system as it
+ * is observed by the process, i.e., the virtualized guest rather than the
+ * underlying host.
```

Comment:
```suggestion
 * underlying host.
 * https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/resource/semantic_conventions/os.md
```

I think it would be nice to have link to spec for convenience

https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-semantic-conventions/src/trace/general.ts#L17-L22

Reply:
Think this would be a nice addition. @obecny feel free to merge after this.